aboutus.html

```diff
@@ -54,13 +55,13 @@ <h1>Gandhi Associates</h1>
       </div>
       <div class="banner">
         <div class="container">
-          <div class="shadow"><img src="img/slider-shadow.png" style="width:100%;heght:auto;"></div>
+          <div class="shadow"><img src="img/slider-shadow.png" alt="Slider" style="width:100%;heght:auto;"></div>
```

Comment:
Alt text must represent what the image is all about, here the **slider** does not fulfill its purpose.